CRM/Civigiftaid/Utils/GiftAid.php

```diff
@@ -215,9 +215,10 @@ static function setDeclaration( $params ) {
         if ( $params['eligible_for_gift_aid'] == 1 ) {
 
             if ( !$currentDeclaration ) {
-                // There is no current declaration so create new.
-               CRM_Civigiftaid_Utils_GiftAid::_insertDeclaration( $params, $endTimestamp );
-
+              // There is no current declaration so create new.
```

Comment:
can you update the inline comments here so it is more clear why you are calling _deleteDecarationWithNullStartDate then _insertDeclaration, the current inline comments does not add any information that I cannot get from reading the code itself, so the comment should explain the "why" instead not the "what".